bedrock/firefox/templates/firefox/products/lockwise.html

```diff
@@ -57,43 +57,44 @@ <h2 class="c-sub-navigation-title">
     {% else %}
       {% set _img = 'img/firefox/lockwise/en.png'%}
     {% endif %}
-    {% call hero(
-      title=ftl('lockwise-firefox-lockwise'),
-      tagline=ftl('lockwise-take-your-passwords-everywhere'),
-      desc=ftl('lockwise-securely-access-the-passwords'),
-      class='mzp-c-hero mzp-has-image lockwise-hero',
+
+    {% call split(
       image_url= _img,
-      include_cta=True,
-      heading_level=1,
+      media_after=True,
+      media_class='mzp-l-split-h-center',
+      block_class='mzp-l-split-center-on-sm-md '
     ) %}
 
+    <h1 class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-lockwise"> {{ ftl('lockwise-firefox-lockwise') }} </h1>
+    <p class="mzp-u-title-md">{{ ftl('lockwise-take-your-passwords-everywhere') }}</p>
```

Comment:
Not related to this change, but semantically this should be an `<h2>` instead of a `<p>`?